### ISSUE TITLE
'Segment Details' and 'Segment Summary' naming, navigation

### ISSUE THREAD

**Issue by EVGrokker** [aesculus/EVTO-App-Feedback#227]
I think you're uncomfortable with the names of these two menu items, and would prefer to see 'Trip Details' and 'Trip Summary'. Frankly, so would I.

The way to do this would be to make it clear within those panel displays that you're currently viewing one of n segments within the trip, and to offer a mechanism to move between the trip segments. 

For **Segment Summary** (to be renamed **Trip Summary**):
The header would also need to show the name of the segment.
One option would be to add another header row below 'Segment Summary' with the name of the segment. This row could have left and right arrow icons at the edges. 

Or, you could enable swiping on that row to move through the segments. With this approach, there should be a visual clue, like appending '(1/2)', '(2/2)' to the segment name.

For **Segment Details** (to be renamed **Trip Details**):
You already allow swiping on the detail area to move between the segments, so arguably you could simply rename this panel **Trip Details** and be done with it. But again, there should be some visual clue that you can move between segments from this screen. The left and right arrows at the edges of the header area where the segment name is displayed would work here as well. Or, appending the segment number as described above.

**Comment by aesculus** [comment]
What do we call the current Trip Segments Summary (ie  the actual segment summary from the map icon).

**Comment by EVGrokker** [comment]
It's referred to by that name, 'Trip Segments Summary' in the help, and that's the name on the dialog header.

**Comment by aesculus** [comment]
OK. Just wanted you to weigh in on the whole rename package. :-)

**Comment by aesculus** [comment]
> The header would also need to show the name of the segment.
One option would be to add another header row below 'Segment Summary' with the name of the segment. This row could have left and right arrow icons at the edges.

Working on this approach. I don't like variable names in headers because the format can get weird when names get long. Plus floating that icon is trouble.

**Comment by EVGrokker** [comment]
Is the header swipable? What about using ⇠ and ⇢ before/after the segment name? Obviously first segment would not have a left arrow prefix, last segment would not have a right arrow prefix.

**Comment by EVGrokker** [comment]
Correction: Last segment would not have a right arrow suffix.

**Comment by aesculus** [comment]
Each line item, including the header, is a list item.  So I have to hack the header to look like a header. So if we go for the swiping/arrow thing it probably does not make much of a difference.

I can do some experiments to see if arrows or swiping is easier. Right now my guess is that swiping might be easier, but would people notice they could swipe, even with arrow indicators?

**Comment by EVGrokker** [comment]
Touching arrows at either end of the header would be more visibly obvious if you can implement it.

Another idea - dots in a visible area at the bottom, like you use in the getting started screen sequence, would clue the user of how many segments there are to swipe between.

**Comment by aesculus** [comment]
> Another idea - dots in a visible area at the bottom, like you use in the getting started screen sequence, would clue the user of how many segments there are to swipe between.

But the bottom could be off the screen if the list is long.  I can't put in a fixed footer. :-(

**Comment by aesculus** [comment]
I have increased the font to be the same size as the list items. I have also made the header fixed (in order to accommodate arrow buttons in the future). V1.1.4 (9)

You will need to find a waypoint list larger than the device can handle to test the scrolling nature.

If you like this one I can change the Trip Segments one too. And I might be able to make the summary data fixed if you feel that it would work for a phone. I am worried the list would only be 5 items long on an iPhone before it had to be scrolled.

**Comment by EVGrokker** [comment]
A couple of observations:
- The increased font size in the header looks good. I'd still like to see a bit more breathing space between the icon and the text.
- I would propose moving the summary data to the top to make it part of the fixed header, so that when the list of waypoints has to scroll, the summary is always visible, along with the segment name and left/right nav controls. I didn't place the nav controls in the mockup. 
- I've mocked up the mileage/drive time/charge time layout I have in mind for the top line of the summary. This would be exactly the same line of data proposed for the Trip Segments dialog, #279.
- It might make sense to put the segment name above the mileage/drive time/charge time line, so that this section is visually identical to what's listed in the Trip Segments list.
- I did add enough waypoints to cause the list to scroll, and there were problems with the last item not being visible when dragging the list up. But rather than chase that now, I propose you make these changes first, then work on the scrolling behavior.

---
![segment summary fixed header copy](https://cloud.githubusercontent.com/assets/26678403/25274038/9aeabbfa-264b-11e7-8dcf-cf867bd0eded.png)
